save-backend/src/main/kotlin/com/saveourtool/save/backend/controllers/DemoManagerController.kt

```diff
@@ -84,20 +87,46 @@ class DemoManagerController(
         @PathVariable organizationName: String,
         @PathVariable projectName: String,
         authentication: Authentication,
-    ): Mono<DemoStatus> = blockingToMono {
-        projectService.findByNameAndOrganizationNameAndStatusIn(projectName, organizationName, setOf(ProjectStatus.CREATED))
-    }
-        .switchIfEmptyToNotFound {
-            "$organizationName/$projectName has no demo linked with it."
-        }
-        .flatMap {
-            lnkProjectGithubService.findByProject(it).toMono()
+    ): Mono<DemoStatus> = getGithubCredentials(organizationName, projectName)
+        .flatMap<DemoStatus?> {
+            webClientDemo.get()
+                .uri("/demo/internal/${it.githubOwner}/${it.githubRepoName}/status")
+                .retrieve()
+                .bodyToMono()
         }
+        .defaultIfEmpty(DemoStatus.NOT_CREATED)
+
+    @GetMapping("/{organizationName}/{projectName}")
+    @RequiresAuthorizationSourceHeader
+    @PreAuthorize("hasRole('ROLE_SUPER_ADMIN')")
+    @Operation(
+        method = "GET",
+        summary = "Get demo info.",
+        description = "Get demo info.",
+    )
+    @ApiResponse(responseCode = "200", description = "Successfully fetched demo status.")
+    fun getDemoInfo(
+        @PathVariable organizationName: String,
+        @PathVariable projectName: String,
+        authentication: Authentication,
+    ): Mono<DemoInfo> = getGithubCredentials(organizationName, projectName)
         .flatMap {
             webClientDemo.get()
                 .uri("/demo/internal/${it.githubOwner}/${it.githubRepoName}")
                 .retrieve()
                 .bodyToMono()
         }
 
+    private fun getGithubCredentials(organizationName: String, projectName: String) = blockingToMono {
+        projectService.findByNameAndOrganizationNameAndStatusIn(projectName, organizationName, setOf(ProjectStatus.CREATED))
+    }
+        .switchIfEmptyToNotFound {
+            "Could not find project $projectName in organization $organizationName"
```

Comment:
```suggestion
            "Could not find project $projectName in organization $organizationName."
```

Reply:
Fixed in https://github.com/saveourtool/save-cloud/pull/1668/commits/148e7f4b03d1740bff59e3796f2e9ce378f8a057